lib/task-data/schemas/install-centos.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "rackhd/schemas/v1/task-schema",
+    "id": "rackhd/schemas/v1/tasks/install-centos",
+    "copyright": "Copyright 2016, EMC, Inc.",
+    "title": "Install CentOS",
+    "description": "The parameters for CentOS installation",
+    "describeJob": "Job.Os.Install",
+    "allOf": [
```

Comment:
@yyscamper Thanks for checking on this.  I agree that it makes sense to keep things consistent with Redfish schemas if they both work equivalently.